client/src/App.jsx

```diff
@@ -19,6 +20,9 @@ const App = () => (
       <Route exact path={'/event/:eventId/:name'}>
         <SingleEventCard />
       </Route>
+      <Route exact path={'/login'}>
```

Comment:
```suggestion
      <Route exact path={'/register'}>
```
Because we have tab component have two components the login and signup